src/test/resources/db/migration/V1__init_db.sql

```diff
@@ -110,25 +110,17 @@ CREATE TABLE machine_annotation_services
     deleted_on                    timestamp with time zone
 );
 
-CREATE
-EXTENSION IF NOT EXISTS "uuid-ossp";
-
-create table mas_job_record
+create table mas_job_record_new
 (
-    job_id         uuid default uuid_generate_v4() not null
-        constraint mas_job_record_pk
+    job_id         text                     not null
+        constraint mas_job_record_new_pk
             primary key,
-    state          text                            not null,
-    creator_id     text                            not null,
-    time_started   timestamp with time zone        not null,
+    job_state      text check ( job_state in ('SCHEDULED', 'FAILED', 'COMPLETED', 'RUNNING') )            not null,
```

Comment:
Just one more comment. Types in postgres can be done with: https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
Oof this is a doozy...so I tried this and I don't think that's possible with Jooq's enum converter. 

Given this enum converter: 

```
<configuration>
 <generator>
   <database>
     <forcedTypes>
       <forcedType>
         <userType>eu.dissco.backend.domain.MasJobState</userType>
         <enumConverter>true</enumConverter>
         <includeExpression>.*\.JOB_STATE</includeExpression>
       </forcedType>
     </forcedTypes>
   </database>
 </generator>
</configuration>
```

What this mapping does is map the enum MasJobState to a varchar, which is sent to the database. **It does not create a custom postgres type that can be used in a java application.** 

I did create a postgres type called JobState, and when I tried it out (both in test and using the actual table), the database rejected the data because jooq was mapping the value of the enum to a varchar and not JobState.

Reply:
It should be possible to do. Now we don't get the advantages of the enum in the database. If you cannot forceType it you could just map it. When you run JOOQ with the enum (without forcedTypes) it should generate Enum classes for you. You would then need to map to these classes (just as the JSONB column).